airbyte-cdk/python/airbyte_cdk/sources/streams/http/requests_native_auth/oauth.py

```diff
@@ -39,6 +42,7 @@ def __init__(
 
         self._token_expiry_date = token_expiry_date or pendulum.now().subtract(days=1)
         self._access_token = None
+        self._connector_config = observe_connector_config(connector_config) if connector_config else None
```

Comment:
I think this won't happen automatically, we need the Authenticator to also update this particular config